Sources/Options.swift

```diff
@@ -869,6 +870,7 @@ public struct FormatOptions: CustomStringConvertible {
                 additionalXCTestSymbols: Set<String> = [],
                 equatableMacro: EquatableMacro = .none,
                 preferFileMacro: Bool = true,
+                lineBetweenConsecutiveGuards: Bool = false,
```

Comment:
I agree that if we make this configurable then `true` will be a better default

```suggestion
                lineBetweenConsecutiveGuards: Bool = true,
```

Reply:
Fixed. Now this option is true by default